src/modules/commander/commander.cpp

```diff
@@ -3114,6 +3117,25 @@ int commander_thread_main(int argc, char *argv[])
 
 		}
 
+		/* Set home position altitude to EKF origin height if home is not set and the EKF has a global origin.
```

Comment:
This looks good, but can it be pushed into commander_set_home_position() so that the logic is centralized and there's only a single publishing location? We should also update the VEHICLE_CMD_DO_SET_HOME to use the same call, but that can be done later (I didn't even know it was implemented).

Reply:
It probably can be centralised. i will have a go at that.